soh/soh/Enhancements/debugger/debugSaveEditor.h

```diff
@@ -500,6 +500,8 @@ const std::vector<FlagTable> flagTables = {
 
         { RAND_INF_KAK_100_GOLD_SKULLTULA_REWARD, "KAK_100_GOLD_SKULLTULA_REWARD" },
         { RAND_INF_GREG_FOUND, "RAND_INF_GREG_FOUND" },
+
+        { RAND_INF_CAUGHT_LOACH, "RAND_INF_CAUGHT_LOACH" },
```

Comment:
Be warned:  this will create a merge conflict since there are other PRs in flight that are adding to this table.